### ISSUE TITLE
Add timestamp inputs for "Focus Mode" start/end

### ISSUE THREAD

**Issue by markerikson** [RecordReplay/devtools#5953]
Currently the only way to set the start and end timestamps of "Focus Mode" is to drag the time range sliders.  This can be imprecise and finicky.

It would be helpful to add a pair of timestamp inputs to the "Edit Focus" modal, which would allow manual / more precise input of the start and end focus range timestamps without having to drag the sliders:

![image](https://user-images.githubusercontent.com/1128784/159958595-5c03ea9e-5a16-49d6-9087-1871a22689ae.png)

**Comment by bhackett1024** [comment]
I agree this is imprecise, and it might be worth thinking out a little further about where we want this feature to evolve.  I think it would be really nice (and I think we've gotten feedback from users about this) if we could set any point in the recording as the beginning/end of the focused range, i.e. anywhere we can pause, including particular points in the script execution.  The backend doesn't have support for this yet, but it would need a different UX than adding values to the modal, like the user would need to jump to the boundaries they want and have a way to mark them as the beginning/end of the focused range.  Once they can do that we'll have more precision with the backend's current range of support, and something to build on as we add the needed backend functionality to make this work as expected in all cases.

**Comment by jasonLaster** [comment]
We're starting some of that here
https://github.com/RecordReplay/devtools/issues/5895

Moving this issue to a discussion so we can discuss further ways to focus
